connect/api/src/main/java/org/apache/kafka/connect/connector/Connector.java

```diff
@@ -43,6 +45,7 @@
 public abstract class Connector {
 
     protected ConnectorContext context;
+    protected static ConfigDef configDef = new ConfigDef();
```

Comment:
Why is this here? Seems weird to have a single, static, shared instance. As far as I can tell, this is being used multiple times by different connectors too, e.g. it is modified by both FileStreamSourceConnector and FileStreamSinkConnector.